### ISSUE TITLE
Consider best ways of presenting examples for multi-language project

### ISSUE THREAD

**Issue by mccalluc** [opendp/opendp#976]
With the R bindings, OpenDP is becoming increasingly multilingual. Maintaining documentation in parallel for different languages will be hard. Some projects let you tab between examples in different languages, so the narrative portions don't need to be duplicated for different languages. Examples:

- [h20.ai](https://docs.h2o.ai/h2o/latest-stable/h2o-docs/cross-validation.html)
- [polars](https://pola-rs.github.io/polars/user-guide/concepts/data-structures/)

Possible roadmap:
- Decide if this is a concern.
- If it is, review different tools that support this style of documentation.
- Looking at what different tools support, ask if this is something we want to use for all docs. (Are there examples of multi-lingual doc tools that can show graphical output?)
- File a new issue for implementation, when path is clear.

See also:
- #971
> Notebooks are also more difficult to:
> * internationalize
> * give code snips in multiple programming languages (like https://docs.h2o.ai/h2o/latest-stable/h2o-docs/cross-validation.html)
>
> Might still be worth it (there may be solutions here), but should consider these tradeoffs before converting.

_Originally posted by @Shoeboxam in https://github.com/opendp/opendp/issues/971#issuecomment-1789109372_

**Comment by mccalluc** [comment]
Filed a new issue that duplicates this one:
- #1052

It has more useful info, so I'll close this.
